middle_auth_client/decorators.py

```diff
@@ -306,7 +308,11 @@ def decorated_function(*args, **kwargs):
                 local_table_id = kwargs.get(table_arg)
 
             if local_table_id is None and dataset is None:
-                return flask.Response("Missing table_id", 400)
+                return make_api_error(400,
+                                      "missing_table_id",
+                                      msg = "Missing table_id",
+                                      data = {'table_id': local_table_id,
```

Comment:
I think data should be removed since both values are None

Reply:
makes sense